challenge_help_me_carry/src/challenge_hmc_functions/hmc_functions.py

```diff
@@ -111,73 +120,77 @@ def execute(self, userdata):
         else:
             return "abort"
 
+
 class GrabItem(smach.State):
+    """
+    The robot's arm is put in carrying bag pose, waiting to receive the bag
+
+    """
     def __init__(self, robot, empty_arm_designator, current_item):
-        smach.State.__init__(self,
-                             outcomes=['succeeded', 'failed', 'timeout'],
-                             input_keys=['target_room_in'],
+        smach.State.__init__(self, outcomes=['succeeded', 'failed', 'timeout'], input_keys=['target_room_in'],
                              output_keys=['target_room_out'])
         self._robot = robot
         self._empty_arm_designator = empty_arm_designator
         self._current_item = current_item
         robot.base.local_planner.cancelCurrentPlan()
 
     def execute(self, userdata):
-
-        handOverHuman = states.HandoverFromHuman(self._robot,
-                                                 self._empty_arm_designator,
-                                                 "current_item",
-                                                 self._current_item,
-                                                 arm_configuration="carrying_bag_pose")
+        handover_human = states.HandoverFromHuman(self._robot, self._empty_arm_designator, "current_item",
+                                                  self._current_item, arm_configuration="carrying_bag_pose")
 
         userdata.target_room_out = userdata.target_room_in
 
-        return handOverHuman.execute()
+        return handover_human.execute()
 
 
 class NavigateToRoom(smach.State):
+    """
+    Navigate to the target room or place
+
+    """
     def __init__(self, robot):
-        smach.State.__init__(self,
-                             outcomes=['unreachable','arrived','goal_not_defined'],
-                             input_keys=['target_room'])
+        smach.State.__init__(self, outcomes=['unreachable', 'arrived', 'goal_not_defined'], input_keys=['target_room'])
         self._robot = robot
         robot.base.local_planner.cancelCurrentPlan()
 
     def execute(self, userdata):
-
         target_waypoint = challenge_knowledge.waypoints[userdata.target_room]['id']
         target_radius = challenge_knowledge.waypoints[userdata.target_room]['radius']
 
-        navigateToWaypoint = states.NavigateToWaypoint(self._robot, ds.EntityByIdDesignator(self._robot, id=target_waypoint), target_radius)
+        navigate_to_waypoint = states.NavigateToWaypoint(self._robot,
+                                                         ds.EntityByIdDesignator(self._robot, id=target_waypoint),
+                                                         target_radius)
 
-        return navigateToWaypoint.execute()
+        return navigate_to_waypoint.execute()
```

Comment:
I think CamelCase is the desired styling here.

Reply:
Ok, I was just following PyCharm's recommendations :)